examples/grpc-secured-unary/grpc_secured_unary_service.out

```diff
@@ -7,5 +7,5 @@
 # Execute the following command to build the 'service' module.
 $ ballerina build service
 
-# Run the service using the following command.
-$ ballerina run target/bin/service.jar
+# Run the service using the following command and passing Ballerina home path as system property.
```

Comment:
```suggestion
# To run the service, execute the below command by passing Ballerina home path as a system property.
```

Reply:
suggestion: To run the service, execute the below command by passing the path to the Ballerina home directory using the `--b7a.home` parameter.